convert2rhel/main.py

```diff
@@ -222,14 +221,7 @@ def _handle_main_exceptions(process_phase, pre_conversion_results=None):
             subscription.update_rhsm_custom_facts()
 
         rollback_changes()
-        if pre_conversion_results is None:
-            loggerinst.info("\nConversion interrupted before analysis of system completed. Report not generated.\n")
-        else:
-            report.summary(
-                pre_conversion_results,
-                include_all_reports=(toolopts.tool_opts.activity == "analysis"),
-                disable_colors=logger_module.should_disable_color_output(),
-            )
+        provide_status_after_rollback(pre_conversion_results, (toolopts.tool_opts.activity == "analysis"))
```

Comment:
```suggestion
        provide_status_after_rollback(pre_conversion_results, include_all_reports=(toolopts.tool_opts.activity == "analysis"))
```
or since it's long
```suggestion
        include_all_reports = toolopts.tool_opts.activity == "analysis"
        provide_status_after_rollback(pre_conversion_results, include_all_reports)
```